tests/management/test_refresh_program_skills.py

```diff
@@ -0,0 +1,208 @@
+
+    def test_missing_arguments(self):
+        """
+        Test missing arguments.
+        """
+        with self.assertRaisesRegex(
+                InvalidCommandOptionsError,
+                'Either program, args_from_database or all argument must be provided.'
+        ):
+            call_command(self.command)
+
+    def test_missing_arguments_from_database_config(self):
+        """
+        Test missing arguments from --args-from-database.
+        """
+        config = RefreshProgramSkillsConfig.get_solo()
+        config.arguments = ''
+        config.save()
+        with self.assertRaisesRegex(InvalidCommandOptionsError, 'Either program or all argument must be provided.'):
+            call_command(self.command, '--args-from-database')
+
+    @responses.activate
+    @mock.patch('taxonomy.management.commands.refresh_program_skills.get_program_metadata_provider')
+    def test_non_existant_program(self, get_program_provider_mock):
+        """
+        Test that command work as expected if program does not exist for a program uuid.
+        """
+        program_uuid = str(uuid4())
+        get_program_provider_mock.return_value = DiscoveryProgramMetadataProvider([])
+
+        with self.assertRaises(ProgramMetadataNotFoundError) as assert_context:
+            call_command(self.command, '--program', program_uuid)
+
+        self.assertEqual(
+            assert_context.exception.args[0],
+            'No program metadata was found for following programs. {}'.format([program_uuid])
+        )
+
+    @responses.activate
+    @mock.patch('taxonomy.management.commands.refresh_program_skills.get_program_metadata_provider')
+    def test_program_without_overview(self, get_program_provider_mock):
+        """
+        Test that command work as expected if program overview does not exist.
+        """
+        self.program_1.overview = ''
+        self.program_1.save()
+        get_program_provider_mock.return_value = DiscoveryProgramMetadataProvider([self.program_1])
+
+        skill = Skill.objects.all()
+        program_skill = ProgramSkill.objects.all()
+        self.assertEqual(skill.count(), 0)
+        self.assertEqual(program_skill.count(), 0)
+
+        with LogCapture(level=logging.INFO) as log_capture:
+            call_command(self.command, '--program', self.program_1.uuid, '--commit')
+            self.assertEqual(len(log_capture.records), 3)
+            messages = [record.msg for record in log_capture.records]
+            self.assertEqual(
+                messages,
+                [
+                    '[TAXONOMY] Refresh Program Skills. Options: [%s]',
+                    '[TAXONOMY] Refresh program skills process started.',
+                    '[TAXONOMY] Refresh %s skills process completed. \n'
+                    'Failures: %s \n'
+                    'Total %s Updated Successfully: %s \n'
+                    'Total %s Skipped: %s \n'
+                    'Total Failures: %s \n'
+                ]
```

Comment:
Same question for %s, shouldn't these be strings here instead of actual %s.

Reply:
same reason as above!